recipes/metview/meta.yaml

```
@@ -0,0 +1,75 @@
+{% set version = "5.6.0" %}
+
+package:
+  name: metview
+  version: {{ version }}
+
+source:
+  url: https://confluence.ecmwf.int/download/attachments/30048389/Metview-{{ version }}-dev3-Source.tar.gz
+  sha256: f796377bb33f9f0afdc9dc4ddd28cae5d8b470d3f1fc1cb0c357334b1b30f153
+  patches:
+    - 0001-Use-find_package-instead-of-find_library-for-threads.patch
+    - 0002-Pass-CPP-to-rpcgen.patch
+    - 0003-Pass-CPP-to-rpcgen.patch
+    - 0005-Remove-cxx11-flags.patch
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+  detect_binary_files_with_prefix: true
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('fortran') }}
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxfixes') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+    - {{ cdt('libxcb') }}  # [linux]
+    - {{ cdt('libxext') }}  # [linux]
+    - {{ cdt('libxau') }}  # [linux]
+  host:
+    - curl
+    - eccodes
+    - magics
+    - boost
+    - qt
+    - flex
+    - bison
+    - libnetcdf
+    - netcdf-cxx4=4.2
```

There needs to be a space here:
```suggestion
    - netcdf-cxx4 =4.2
```

---

@chrisburr thanks for fixing & merging!

Could you explain why a space is needed? Why did the automatic linter not catch the above code if it's a problem?

---

So far as I know the specification requires there to be a space but it's only enforced in some parts of `conda-build`. Creating feedstocks after recipes are merged in `staged-recipes` often fails if it's not included.

It's something that should be added to the linter at some point.

---

Thanks for the info :)